src/components/Main.ui.js

```diff
@@ -366,19 +366,6 @@ export class MainComponent extends React.Component<Props> {
                 renderRightButton={<SideMenuButton />}
               />
 
-              <Scene
-                key={SCAN}
-                navTransparent
-                onEnter={props => {
-                  this.props.requestPermission('camera')
-                  this.props.dispatchEnableScan()
-                }}
-                onExit={this.props.dispatchDisableScan}
-                component={ifLoggedIn(ScanScene)}
-                renderLeftButton={<BackButton onPress={this.handleBack} />}
-                renderRightButton={<SideMenuButton />}
-              />
-
```

Comment:
Why is this removed? I want more context to this.

Reply:
@samholmes because it caused build error and not used, also we don't have `SCAN` const. and could you explain why it was added here - 418baae83494db8790f71872af7162e22d3cec25 ? I assume it was accidentally

Reply:
I don't have the context as to where it is there if it's not used. I'll have to punt this question to other members of the team to determine if it's the right thing to remove.

Reply:
Resolved: It was an artifact of rebasing the old/original FIO Staking Scenes PR. It should be removed along with a couple of other scenes (fixed here https://github.com/EdgeApp/edge-react-gui/pull/3118).